kaggle_environments/envs/football/test_football.py

```diff
@@ -0,0 +1,233 @@
+from kaggle_environments import make, evaluate
+from kaggle_environments.errors import DeadlineExceeded
+import copy
+
+env = None
+
+
+def before_each(state=None, configuration=None):
+    global env
+    steps = [] if state == None else [state]
+    env = make("football", steps=steps,
+               configuration=configuration, debug=True)
+
+
+def test_to_json():
+    before_each()
+    json = env.toJSON()
+    assert json["name"] == "football"
+    assert json["rewards"] == [0.0, 0.0]
+    assert json["statuses"] == ["ACTIVE", "ACTIVE"]
+    assert json["specification"]["reward"]["type"] == ["number", "null"]
+
+def clear_minimap(state):
+  state = copy.deepcopy(state)
+  for entry in state:
+    entry.observation.minimap = []
+  return state
+
+
+def test_single_agent():
```

Comment:
Solid start to testing. I'm guessing there are a number of edge cases about how units collide, out of bounds, etc that are worth testing as well. Is there another test suite for research football that already covers these cases?

Reply:
Yes, the environment has its own test suite.

Here, I wanted to only focus on testing the 'integration' part.
And yes, I plan to add more tests as we go.